CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Versions
 
+## 1.0.10
```

Comment:
For next time, normally in our plugins with changelogs, we break out the version bumps. See React Native repo for example.